src/screens/Sidebar/CannyBoard/component.js

```diff
@@ -19,24 +20,26 @@ class Board extends React.Component {
     this.getCannySsoToken();
   }
 
-  getCannySsoToken = () => {
+  getCannySsoToken = async () => {
     const accessToken = this.props.navigation.state.params.token;
-    fetch('https://kitsu.io/api/edge/sso/canny', {
-      method: 'GET',
-      headers: {
-        Accept: 'application/vnd.api+json',
-        Authorization: `Bearer ${accessToken}`,
-      },
-    })
-    .then(response => response.json())
-    .then((responseJson) => {
-      if (responseJson && responseJson.token) {
+    try {
+      const response = await fetch('https://kitsu.io/api/edge/sso/canny', {
+        method: 'GET',
+        headers: {
+          Accept: 'application/vnd.api+json',
+          Authorization: `Bearer ${accessToken}`,
+        },
+      });
+      const json = await response.json();
+      if (json && json.token) {
         this.setState({
-          ssoToken: responseJson.token,
+          ssoToken: json.token,
           loading: false,
         });
       }
-    });
+    } catch (e) {
```

Comment:
We are probably going to want to go thru and look at our `catch` blocks. Chances are they aren't being sent to sentry when we might want some of them to.